helm/botkube/values.yaml

```diff
@@ -1187,6 +1187,13 @@ plugins:
     enabled: true
     port: 2115
     targetPort: 2115
+    # -- Botkube Restart Policy on plugin failure.
+  restartPolicy:
```

Comment:
So, to better understand: It will do the heatlhcheck each 10s and if it restarts 5 times, the plugin will be disabled with this setup, am I correct?

Reply:
exactly, however I think that we can bump the disable threshold to ~10 because for now it's for the whole life of the plugin, until the botkube pod won't be restarted. I would like to change it to but probably in the follow-up PRs.